pallets/ethereum-transactions/src/tests/tests.rs

```diff
@@ -243,10 +240,7 @@ mod reserve_transaction_id {
 
         #[test]
         fn fails_if_transaction_already_reserved() {
-            let mut ext = ExtBuilder::build_default()
-                .with_genesis_config()
-                .with_validators()
-                .as_externality();
+            let mut ext = ExtBuilder::build_default().with_validators().as_externality();
```

Comment:
Why? I do like this pipeline style, where each line shows a different operation in a clear and much more visible way.

Reply:
This is a personal preference, but I don't feel the need to save on number of lines and instead write longer lines.
We're in an age where we don't print code any more, so it's not about saving paper. In that sense, I don't find it a problem if files are longer, but I sure do find a problem if I have to scroll left and right, as well as up and down. 

And that said, even if I don't have to scroll left and right, it costs me mental energy if I have to parse even a short line to find where the breaking points are. If the code layout could tell me that immediately, it would save me energy and keep me productive for longer in the day.

Reply:
I think this may be related to the code formatting linting checks as I have now removed the with_genesis_config(). Will double check.